lib/client/session_windows.go

```diff
@@ -45,22 +230,585 @@ func newSession(
 	legacyID bool,
 	enableEscapeSequences bool,
 ) (*NodeSession, error) {
+	// The conpty library blindly attempts to load an execute Windows syscalls
+	// that are only present in Windows 10 v1809 or later, and will cause a
+	// panic if they do not exist. Check first to ensure older Windows releases
+	// exit with an informative error message.
+	err := ensureSyscallsExist()
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	cpty, err := conpty.New(teleport.DefaultTerminalWidth, teleport.DefaultTerminalHeight)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	if stdin == nil {
+		stdin = cpty.InPipe()
+	}
+	if stdout == nil {
+		stdin = cpty.OutPipe()
```

Comment:
On this week's episode of "How Did This Ever Work?", I learn that conpty is entirely unnecessary for client-side SSH, and only noticed after realizing that this code has been super broken the whole time because of a typo.

... I'll be removing the conpty library along with the refactoring work. Will need to retest some things since we may now be compatible out of the box with older Win10 releases.